src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -373,7 +373,7 @@ static function ( $event ) use ( $date_object ) {
 	/**
 	 * {@inheritDoc}
 	 */
-	protected function calculate_grid_start_end( $date ) {
+	public function calculate_grid_start_end( $date ) {
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Public method name "Month_View::calculate_grid_start_end" is not in camel caps format
* :no_entry_sign: Opening brace should be on a new line

<sub>hash: 8f106bb783befbb55c96d09a9fc9cfaa</sub>